src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -51,6 +53,7 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
         placeholder,
         readOnly: readonly,
         onBlur: () => onBlur && onBlur(JSON.stringify(localValue.current)),
+        className: `tw-p-${padding}`,
```

Comment:
this won't work in production mode, since it get's purged: see https://v2.tailwindcss.com/docs/just-in-time-mode "Dynamic values"